google-translate-connector/config/variables.yaml

```diff
@@ -0,0 +1,10 @@
+# == Variables ==
+# 
+# You can define here your project Variables.
+# If you want to define/override a Variable for a specific Environment, 
+# add an additional ‘variables.yaml’ file in a subdirectory in the ‘Config’ folder: 
+# '<project>/Config/_<environment>/variables.yaml
+#
+Variables:
+#  myVariable: value
+  apiKey: ''
```

Comment:
please add the `[password]` hint, so that the variable will be encrypted and hidden front-ends, such as the engine-cockpit.
also, the key should be different, having a parent with the connector name `google-translate`
https://dev.axonivy.com/doc/10.0/designer-guide/configuration/variables.html